### ISSUE TITLE
Rule proposal: prop quotes

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#12]
Similar to ESLint's built-in [quotes](http://eslint.org/docs/rules/quotes.html) rule, it would be nice to have a rule that enforces the style of quotes used for props. For instance, if set to "double":

Good:

```jsx
<MyComponent myProp="value" />
```

Bad:

```jsx
<MyComponent myProp='value' />
```

**Comment by lo1tuma** [comment]
As mentioned in https://github.com/eslint/eslint/issues/2011, the suggest rule has nothing to do with react. It is a rule specific to the jsx syntax.

**Comment by xjamundx** [comment]
JSX is just a short-hand for writing react. This plugin already has several other JSX-related rules. I'm not following the logic.

**Comment by oliviertassinari** [comment]
Shouldn't we remove this rule? https://github.com/eslint/eslint/commit/3988b47a4b17fc1277c0956e2010d07d1e665cc5
